src/TrackingOptOut.php

```diff
@@ -101,30 +124,23 @@ public static function registerOptOutShortcode ($atts, $content = NULL) {
     }
   }
 
-  /**
-   * Creates a submenu page in the WordPress admin "Settings" section.
-   *
-   * @implements admin_menu
-   */
-  public static function admin_menu() {
-    add_submenu_page('options-general.php', __('WordPress Core Standards Settings', Plugin::L10N), __('WordPress Core Standards', Plugin::L10N), 'manage_options', 'core-standards-settings', [__CLASS__, 'renderSettings']);
-  }
-
   /**
    * Renders the settings page template.
    */
   public static function renderSettings() {
     Plugin::renderTemplate(['templates/settings.php']);
   }
 
+  public static function enqueueScriptsFirstHead() {
+    wp_enqueue_script(Plugin::PREFIX . '-ga-opt-out', Plugin::getBaseUrl() . '/dist/scripts/ga-opt-out.js');
+  }
+
   /**
    * @implements wp_enqueue_scripts.
    */
   public static function wp_enqueue_scripts() {
-    if (get_option('core_standards_optout_active')) {
-      wp_enqueue_script(Plugin::PREFIX . '-opt-out', Plugin::getBaseUrl() . '/dist/scripts/opt-out.js', ['jquery'], FALSE, TRUE);
-      wp_localize_script(Plugin::PREFIX . '-opt-out', 'core_standards', ['optout_confirmation_message' => __('Google Analytics tracking has been disabled.', Plugin::L10N)]);
-    }
+    wp_enqueue_script(Plugin::PREFIX . '-opt-out', Plugin::getBaseUrl() . '/dist/scripts/opt-out.js', ['jquery'], FALSE, TRUE);
```

Comment:
1. `enqueueScriptsFirstHead()` above uses `-ga-opt-out` instead of `-opt-out` – typo or intended?  If it's intended, then the difference between both is not clear at all and they should have clearer names.

2. Some code uses `optout` (varnames, option names, etc) instead of `opt-out` (both in identifier and filename).  That's not good for discovery / search purposes.  Can you make that consistent?  (Note that the class name `...OptOut` is converted into `...opt_out` with `uncamelize()`)